lib/inc/drogon/HttpAppFramework.h

```diff
@@ -745,6 +745,21 @@ class DROGON_EXPORT HttpAppFramework : public trantor::NonCopyable
      */
     virtual HttpAppFramework &disableSession() = 0;
 
+    /// Enable saving of anonymous sessions.
+    /**
+     *
+     * @note
+     * Saving of anonymous sessions is disabled by default
+     */
+    virtual HttpAppFramework &enableSaveAnonSession() = 0;
+
+    /// Disable saving of anonymous sessions.
+    /**
+     * @note
+     * Saving of anonymous sessions is disabled by default
+     */
+    virtual HttpAppFramework &disableSaveAnonSession() = 0;
```

Comment:
For consistency with older users, I think we should enable this feature by default, otherwise some applications will not work after upgrading drogon.

Reply:
Ah yes, I think all declarations are enabled by default. Will double check and change the description.